jupyter_book/book_template/_sass/components/_components.textbook__sidebar-right.scss

```
@@ -26,6 +30,10 @@ $right-sidebar-center-offset: ($page-max-width - $sidebar-width) / 2 -
         top: 3 * $spacing-unit;
         right: 0; /* [1] */
         padding-right: $spacing-unit;
+        overflow-y: scroll;

```

```suggestion
        overflow-y: auto;
```

I believe if we set this to "auto" then the scrollbar will only show up when it's needed, but not when the page is big enough to contain the whole bar. What do you think?